docs/docsite/rst/community/other_tools_and_programs.rst

```diff
@@ -19,19 +19,21 @@ Popular editors
 Emacs
 -----
 
-A free, open-source text editor and IDE that supports auto-indentation, syntax highlighting and built in terminal shell(among other things).
+A free, open-source text editor and IDE that supports auto-indentation, syntax highlighting and a built in terminal shell (among other things).
 
 * `yaml-mode <https://github.com/yoshiki/yaml-mode>`_ - YAML highlighting and syntax checking.
 * `jinja2-mode <https://github.com/paradoxxxzero/jinja2-mode>`_ - Jinja2 highlighting and syntax checking.
-* `magit-mode <https://github.com/magit/magit>`_ -  Git porcelain within Emacs.
+* `magit-mode <https://github.com/magit/magit>`_ - Git porcelain within Emacs.
 * `lsp-mode <https://emacs-lsp.github.io/lsp-mode/page/lsp-ansible/>`_ - Ansible syntax highlighting, auto-completion and diagnostics.
 
 
 PyCharm
 -------
 
-A full IDE (integrated development environment) for Python software development. It ships with everything you need to write python scripts and complete software, including support for YAML syntax highlighting. It's a little overkill for writing roles/playbooks, but it can be a very useful tool if you write modules and submit code for Ansible. Can be used to debug ``ansible-core``. For more information, see `PyCharm <https://www.jetbrains.com/pycharm/>`_
+A full IDE for Python software development. It ships with everything you need to write Python scripts and applications, including support for YAML syntax highlighting. It can be a very useful tool if you write modules and submit code for Ansible. Can be used to debug ``ansible-core``. For more information, see `PyCharm <https://www.jetbrains.com/pycharm/>`_. Useful plugins include:
 
+* `Ansible Lint <https://plugins.jetbrains.com/plugin/20905-ansible-lint>`_ - Ansible Lint integration with automatic/continuous annotation of errors, warnings, and info while editing.
+* `Ansible Vault Integration <https://plugins.jetbrains.com/plugin/14353-ansible-vault-integration>`_ - Integrate the ansible vault directly into IntelliJ IDEA with context actions for vaulting and unvaulting secrets.
```

Comment:
This plugin is licensed under [a non-free license](https://github.com/timo-reymann/idea-ansible-vault-integration/blob/master/LICENSE). I would add something like

```suggestion
* `Ansible Vault Integration <https://plugins.jetbrains.com/plugin/14353-ansible-vault-integration>`_ - Integrate the ansible vault directly into IntelliJ IDEA with context actions for vaulting and unvaulting secrets. Note that this plugin is non-free software.
```

Reply:
The licencse states:

> Permission is hereby granted, free of charge, to any person obtaining a copy of
> this software and associated documentation files (the "Software"), to deal in
> the Software without limitation in the rights to use, copy, modify, merge,
> publish, distribute, sublicense, and/or sell copies of the Software, and to
> permit persons to whom the Software is furnished to do so, subject to the
> following conditions:

Which part do you consider "non-free"?

Reply:
The paragraphs

> The Software may not be used in applications and services that are used for or
aid in the exploration, extraction, refinement, processing, or transportation
of fossil fuels.
>
> The Software may not be used by companies that rely on fossil fuel extraction
as their primary means of revenue. This includes but is not limited to the
companies listed at https://climatestrike.software/blocklist

in the license add restrictions on who can use the software. That makes the license non-free.

Reply:
Heya all, 
@4ch1m contacted me about the issue and I changed the licence to Apache 2.0.
Therefore, is fully free now and matches the ideology of open source more closely.

Cheers

Reply:
:+1: